tests_e2e/orchestrator/scripts/get-agent-pythonpath

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
this script can be used to set the PYTHONPATH for any Python (this PR uses it for Pypy) before invoking 'waagent' or importing any of its modules